WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/AccountSettingsAnalyticsTracker.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.ui.prefs.accountsettings
+
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.SETTINGS_DID_CHANGE
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.EMAIL_CHANGE
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.PASSWORD_CHANGE
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.PRIMARY_SITE_CHANGE
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.USERNAME_CHANGE
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.USERNAME_CHANGE_SCREEN_DISMISSED
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.USERNAME_CHANGE_SCREEN_DISPLAYED
+import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsAction.WEB_ADDRESS_CHANGE
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import javax.inject.Inject
+
+private const val SOURCE = "source"
+private const val SOURCE_ACCOUNT_SETTINGS = "account_settings"
+private const val TRACK_PROPERTY_FIELD_NAME = "field_name"
+private const val TRACK_PROPERTY_PAGE = "page"
+private const val TRACK_PROPERTY_PAGE_ACCOUNT_SETTINGS = "account_settings"
+
+enum class AccountSettingsAction(val trackProperty: String? = null) {
+    EMAIL_CHANGE("email"),
+    PRIMARY_SITE_CHANGE("primary_site"),
+    WEB_ADDRESS_CHANGE("web_address"),
+    PASSWORD_CHANGE("password"),
+    USERNAME_CHANGE("username"),
+    USERNAME_CHANGE_SCREEN_DISPLAYED,
+    USERNAME_CHANGE_SCREEN_DISMISSED
+}
```

Comment:
@SowmiaS Thanks for moving the tracking logic to another class. I am just not sure about the Action Name that you have used. Would `EMAIL_CHANGED, PRIMARY_SITE_CHANGED` be a better name to keep consistency with the `USERNAME_CHANGE_SCREEN_DISPLAYED` pattern(Past tense)?. Also, `USERNAME_CHANGE_SCREEN_DISPLAYED` is not exactly an Action per se. How about we change the class name to `AccountSettingsEvent` or something like that?. 🤔